e2e/libs/sharemanager/crd.py

```
@@ -16,3 +20,49 @@ def list(self, label_selector=None):
+    def delete(self, name):
+        logging(f"deleting sharemanager {name} ...")
+        return self.obj_api.delete_namespaced_custom_object(
+            group="longhorn.io",
+            version="v1beta2",
+            namespace="longhorn-system",
+            plural="sharemanagers",
+            name=name
+        )
+    
+    def wait_for_running(self, name):
+        for i in range(self.retry_count):
```

Ruff

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)